lib/hystrix-formatter.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+// A function to map our stats data to the hystrix format
+// returns JSON
+function hystrixFormatter (stats) {
+  const json = {};
+  json.type = 'HystrixCommand';
+  json.name = 'ola';
```

Comment:
I'm not sure if i can get access to this information currently.  Since the Status object emits the snapshot event,  i only get the stats data during that event.   maybe the Circuit class should emit the event(which i'm not sure about), or we add some metadata to the stats object that gets emitted during that snapshot event